tool/js/printTWASApps.js

```diff
@@ -0,0 +1,31 @@
+/*****************************************************************
```

Comment:
Did you mean to include this script? Seems like only apps.js and comps.js are used.

Reply:
Good catch. No, that does not belong. I removed it.